<?xml version="1.0"?>
<launch>
    <node pkg="waypoint_updater" type="waypoint_updater.py" name="waypoint_updater">
        <param name="final_waypoints_pub_queue" value="1" />
        <param name="rate" value="50" />
        <param name="lookahead_waypoints" value="30" />
        <param name="max_deceleration" value="0.5" />
    </node>
</launch>
